lib/records/ccda.rb

```diff
@@ -114,6 +116,36 @@ def self.procedure(procedure, ccda_record)
           "end_time" => time.to_i + 15.minutes,
         })
       end
+
+      def self.careplans(plan, ccda_record)
+        type = plan['type']
+        time = plan['time']
+        entries = [type] + plan['activities']
+        entries.each do |entry| 
+          care_goal = CareGoal.new({
+            'codes' => CAREPLAN_LOOKUP[entry][:codes],
+            'description' => CAREPLAN_LOOKUP[entry][:description],
+            'start_time' => time.to_i,
+            'status' => plan['status'],
+            'reason' => COND_LOOKUP[plan['reason']]
+          })
+          care_goal['end_time'] = plan['stop'].to_i if plan['stop']
+          ccda_record.care_goals << care_goal
+        end
+      end
+
+      def self.medications(prescription, ccda_record)
+        type = prescription['type']
+        time = prescription['time']
+        medication = Medication.new({
+          "codes" =>  MEDICATION_LOOKUP[type][:codes],
+          "description" => MEDICATION_LOOKUP[type][:description],
+          "start_time" => time.to_i,
+          'reason' => COND_LOOKUP[prescription['reason']]
+        })
```

Comment:
@jawalonoski was there a reason that there isn't a `status` field in the medications record hash?